dts/xtensa/intel/intel_adsp_ace20_lnl.dtsi

```diff
@@ -667,4 +687,52 @@
 			reg = <0x12>;
 		};
 	};
+
+	uaols {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		uaol13: uaol@d {
+			compatible = "intel,uaol-dai";
+			reg = <0x0d>;
+			link = <0>;
+			stream = <0>;
+			status = "okay";
+		};
+		uaol14: uaol@e {
+			compatible = "intel,uaol-dai";
+			reg = <0x0e>;
+			link = <0>;
+			stream = <1>;
+			status = "okay";
+		};
+		uaol15: uaol@f {
```

Comment:
How would you name the node if the SoC contains multiple 'link' devices and they all contain a DAI which have reg = <0x0f> ?
It is better to use label which can scale later?

Reply:
DAI `reg` entries correspond to `ipc4_connector_node_id::v_index`, so I think they are unique at system level and should not duplicate even for multiple UAOL IP instances. However redesigned labels of nodes to reflect their link and stream index, e.g. `uaol01:` denoting link=0 and stream=1.